frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -334,7 +334,7 @@ end
 local highlight_style = {
     {_("Lighten"), "lighten"},
     {_("Underline"), "underscore"},
-    {_("Strikeout"), "strikeout"},
+    {_("Strike through"), "strikeout"},
```

Comment:
To make it more consistent I think it would also be better to change the occurrences of "strikeout" to "strikethrough"

Reply:
I don't read the whole context here but it also seems like _("Strike through") would be better to be _("Strikethrough" more like a noun than a verb?

Reply:
Yes, strikethrough text formatting is also what e.g. Word calls it.

Reply:
```suggestion
    {_("Strikethrough"), "strikeout"},
```

Reply:
👍